hw/ip/otbn/data/otbn.hjson

```diff
@@ -102,30 +102,76 @@
         }
       ]
     } // register : status
-    { name: "ERR_CODE",
-      desc: "Error Code",
+    { name: "ERR_BITS",
+      desc: '''
+        Error bitfield, reads as non-zero if an error was seen during OTBN
+        operation
```

Comment:
Nit: The grammar's not quite right here. The smallest change to fix things is to replace the comma with a full stop and capitalize "reads".